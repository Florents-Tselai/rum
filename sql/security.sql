-- Check security CVE-2020-14350
CREATE FUNCTION rum_numeric_cmp(numeric, numeric) RETURNS int4 AS $$ SELECT 0 $$ LANGUAGE SQL;
CREATE EXTENSION rum;
DROP FUNCTION rum_numeric_cmp(numeric, numeric);

