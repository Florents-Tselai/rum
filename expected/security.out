-- Check security CVE-2020-14350
CREATE FUNCTION rum_anyarray_similar(anyarray,anyarray) RETURNS bool AS $$ SELECT false $$ LANGUAGE SQL;
CREATE EXTENSION rum;
ERROR:  function "rum_anyarray_similar" already exists with same argument types
DROP FUNCTION rum_anyarray_similar(anyarray,anyarray);
